app/src/pages/explorer/parameterized-query/parameterized-query-page.vue

```diff
@@ -171,6 +172,11 @@ export default {
     },
     newQuery () {
       return this.query !== this.lastRunQuery
+    },
+    pageTitle () {
+      const selectionsKeys = Object.keys(this.varSelections)
+      const title = this.varSelections[selectionsKeys[3]]
+      return title[0].toUpperCase() + title.substring(1)
```

Comment:
I'm running into some issues with this change locally. It works fine for me for the first two queries but gives errors on some of the others that have fewer selection keys

Reply:
@aswallace please pull the changes again and we can sync up to discuss